pages/resources.js

```diff
@@ -1,15 +1,160 @@
+import Link from "next/link";
+import { useState } from "react";
 import styled from "styled-components";
 
 const Resources = () => {
   return (
     <div>
       <Headline>Hilfestellen und Beratungsangebote</Headline>
       <List>
-        <ListItem>Fachberatung1</ListItem>
-        <ListItem>Fachberatung2</ListItem>
-        <ListItem>Fachberatung3</ListItem>
-        <ListItem>Fachberatung4</ListItem>
-        <ListItem>Fachberatung5</ListItem>
+        <ListItem>
```

Comment:
this would be a good opportunity to use an additional component! You could add all your styling in the list component, ending up in much cleaner code :)